### PR TITLE
initial code review

### PR DESCRIPTION
**Pull request by alexjohnmartin**

### CODE REVIEW

File: config.qa.js [hunk 1]
```diff
@@ -0,0 +1,17 @@
+var config = {}
+
+config.maxMins = 60 * 24 * 7; //We only allow scheduling of updates up to 7 days in the future
+config.minsUntilMessageShouldNotBeSent = 60; //don't send messages if they have been waiting to send for over X mintes
+config.hoursUntilMessageShouldBeDeleted = 24; //remove messages from the DB once they are X hours past their send time
+
+config.raygunKey = 't2QkxRnuaQM3a3pcWVIXPw==';
+
+config.testPhoneNumber = '12345678';
+config.messagingHost = 'internal.verrus.net';
```
Comment (jskrepnek) [inline config.qa.js]: internal.qa.verrus.net?

File: availability.js [hunk 2]
```diff
@@ -0,0 +1,88 @@
...
+var src = 'http://www.mspairport.com/data/mspparkingstatus.txt';
+var message = '';
+var options = {
+    protocol: 'http',
+    host: 'www.mspairport.com',
+    path: '/data/mspparkingstatus.txt',
+    method: 'GET'
+};
+
+function getParkingAvailability() {
```
Comment (jskrepnek) [inline availability.js]: getParkingAvailability should return the message ... or should it be named calculateParkingAvailability?

File: availability.js [hunk 3]
```diff
@@ -0,0 +1,88 @@
...
+	    console.log(err);
+	});
+}
+
+function parseAvailability(content) {
+	var lindberg = 0;
+	var humphrey = 0;
+	var quick = 0;
+	var lines = content.split('\n');
+	for(var l = 1; l < lines.length; l++) {
```
Comment (jskrepnek) [inline availability.js]: if it's a one based index, then should you go to l <= lines.length?

File: availability.js [hunk 4]
```diff
@@ -0,0 +1,88 @@
...
+	    }
+	});
+
+	req.end();
+	req.on('error', function(err) {
+	    console.log(err);
+	});
+}
+
+function parseAvailability(content) {
```
Comment (jskrepnek) [inline availability.js]: there's no error handling in here ... a poorly formatted content will cause chaos

**Comment by alexjohnmartin** [comment]
Have addressed all the issues raised in the review.

**Pull request closed without merging**